database/dbConnection.js

```diff
@@ -1,5 +1,9 @@
 var mongoose = require('mongoose');
-mongoose.connect('mongodb://localhost:27017/SDC') // change to db for docker purpose. // change back to localhost if using localhost
+require('dotenv').config()
+console.log(process.env) // remove this after you've confirmed it working
```

Comment:
I noticed this is still here. If it's working, looks like you've made a note to remove it from the console.

Reply:
Oh yes i was thinking of adding ip address of EC2 database into .env, so that if i stop and start i just adjust .env file.